langchain-interpreter/src/langchain_interpreter/interfaces/webhook.py

```
@@ -0,0 +1,629 @@
+        secret: The secret string, possibly containing ${env:VAR} syntax.
+
+    Returns:
+        The resolved secret value, or None if not set.
+    """
+    if not secret:
+        return None
+
+    # Use resolve_variables to handle ${env:...} substitution
+    try:
+        resolved = resolve_variables(secret)
+        return resolved
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)